pages/overview/casestudies/_index.md

```diff
@@ -0,0 +1,4 @@
+../../../_includes/community/programs/casestudies/casestudies.md
```

Comment:
Please remove this md file. Let's put all the content in the tsx page.

Reply:
Working on it :)